src/devtools/client/debugger/src/reducers/pause.ts

```diff
@@ -129,7 +130,7 @@ export const executeCommandOperation = createAsyncThunk<
   const symbols = sourceId ? await sourceOutlineCache.readAsync(replayClient, sourceId) : undefined;
   const nextPoint = await getResumePoint(replayClient, state, command);
 
-  const resp = await ThreadFront[command]({
+  const resp = await resumeOperations[command](replayClient, {
```

Comment:
Pleasantly surprised to see that TypeScript covers this without falling back to an `any` type  :)